README.md

```diff
@@ -332,6 +332,32 @@ Param    | Type     | Details
 **name** | *String* | The name of the value.  Must be unique to each Bottle instance.
 **val**  | *Mixed*  | A value that will be defined as enumerable, but not writable.
 
+#### instanceProvider(name, Factory)
+
+Used to register a service instance provider that will return a newly created, fully configured
+instance when called.
+
+Param       | Type       | Details
+:-----------|:-----------|:--------
+**name**    | *String*   | The name of the service.  Must be unique to each Bottle instance.
+**Factory** | *Function* | A function that should return the a fully configured service object.  This factory method will be called whenever a new instance is created.  Gets passed an instance of the container to allow dependency injection when creating a new instance of the service.
```

Comment:
Small typo.  Should read:

A function that should return a fully configured service object.  This factory function will be called whenever a new instance is created.  Gets passed an instance of the container to allow dependency injection when creating a new instance of the service.